t/t2800-jobs-instance-info.t

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+test_description='Test flux jobs command with instance.* attributes'
+
+. $(dirname $0)/sharness.sh
+
+test_under_flux 4 job
+
+export FLUX_URI_RESOLVE_LOCAL=t
+waitfile="${SHARNESS_TEST_SRCDIR}/scripts/waitfile.lua"
+
+flux getattr broker.pid
+
+#  Launch some test instance jobs with varying characteristics
```

Comment:
like prior PR, perhaps a comment on what you're launching, since it can take a bit to grok